### PR TITLE
Post object extension

### PR DESCRIPTION
**Pull request by danthecodingman**

### CODE REVIEW

File: fbuapp/Views/PostTableView.m [hunk 1]
```diff
@@ -119,14 +119,10 @@ - (UITableViewCell *)tableView:(UITableView *)tableView
...
-    [[NSNotificationCenter defaultCenter] addObserverForName:(@"informationSaved") object:(nil) queue:(nil) usingBlock:^(NSNotification * _Nonnull note) {
-        cell.eventAuthor.text = [PFQuery getUserObjectWithId: post[@"eventAuthor"]][@"firstName"];
-
-    }];
-
-    cell.fbProfilePhoto.profileID = [PFQuery getUserObjectWithId: post.eventAuthor][@"fbUserId"];
-    
+    if(post.authorName != nil){//there are pots in database before
+        cell.eventAuthor.text = post.authorName;
+        cell.fbProfilePhoto.profileID = post.authorName;
```
Comment (eazheng) [inline fbuapp/Views/PostTableView.m]: typo

File: fbuapp/Views/PostTableView.m [hunk 2]
```diff
@@ -119,14 +119,10 @@ - (UITableViewCell *)tableView:(UITableView *)tableView
...
     cell.currentUserId = self.currentUserId;
-    cell.eventAuthor.text = [PFQuery getUserObjectWithId: post[@"eventAuthor"]][@"firstName"];
-    [[NSNotificationCenter defaultCenter] addObserverForName:(@"informationSaved") object:(nil) queue:(nil) usingBlock:^(NSNotification * _Nonnull note) {
-        cell.eventAuthor.text = [PFQuery getUserObjectWithId: post[@"eventAuthor"]][@"firstName"];
-
-    }];
-
-    cell.fbProfilePhoto.profileID = [PFQuery getUserObjectWithId: post.eventAuthor][@"fbUserId"];
-    
+    if(post.authorName != nil){//there are pots in database before
```
Comment (eazheng) [inline fbuapp/Views/PostTableView.m]: clarify this comment

File: fbuapp/Views/PillCell.m [hunk 3]
```diff
@@ -22,5 +22,6 @@ -(void)setSelected:(BOOL)selected {
     [selectedBackgroundViewForCell setBackgroundColor:[UIColor lightGrayColor]];
 
     self.selectedBackgroundView = selectedBackgroundViewForCell;
+
```
Comment (eazheng) [inline fbuapp/Views/PillCell.m:25]: extra spacing

**Review by belchercd — COMMENTED** [review]
Good

**Pull request merged**